pkg/kubelet/cm/memorymanager/policy_static.go

```diff
@@ -196,6 +266,13 @@ func (p *staticPolicy) RemoveContainer(s state.State, podUID string, containerNa
 	// Mutate machine memory state to update free and reserved memory
 	machineState := s.GetMachineState()
 	for _, b := range blocks {
+		// This container reused some resources and they must be released to the reusable pool
+		if b.Reused != 0 {
```

Comment:
I unsure we need it, if the container crashed, the allocation should be regenerated from the state file

Reply:
Removed logic from `RemoveContainer()`